import unittest
from moves import MoveValidator
from exception import MoveException

class TestMoveValidator(unittest.TestCase):
    def test_valid_move_validator(self):
        pass 

